presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaConnectorConfig.java

```diff
@@ -69,6 +63,38 @@
      */
     private boolean hideInternalColumns = true;
 
+    /**
+     * The Zookeeper URI.
+     * Example: hostAddress:2181
+     */
+    private String kafkaZookeeperUri = "sdzookeeper.smf1.twitter.com:2181";
+
+    /**
+     * The Zookeeper path for Kafka services.
+     * Example: /twitter/service/kafka/prod/tfe
+     */
+    private String kafkaServicePath = "/twitter/service/kafka/prod/tfe";
```

Comment:
Can these two be configurable rather than hard-coded?

Reply:
Those are default values. They are configurable through deployment scripts, in our case, aurora scripts.

Reply:
For example, we can do follows:

kafka.max.poll.records=5000
kafka.max.partition.fetch.bytes=1048576
kafka.zookeeper.path={{zk_path}}
kafka.zookeeper.uri={{zk_uri}}
kafka.table-names={{topic_name}}
kafka.table-description-dir=etc/kafka/
kafka.hide-internal-columns=false